tray/monitor.go

```diff
@@ -177,9 +177,9 @@ func (ti *Instance) updateSettings() bool {
 
 func (ti *Instance) updateAccountInfo() bool {
 	changed := false
-	loggedIn := false
-	vpnActive := false
-	accountName := ""
+	loggedIn := ti.state.loggedIn
+	vpnActive := ti.state.vpnActive
+	accountName := ti.state.accountName
 
 	payload, err := ti.client.AccountInfo(context.Background(), &pb.Empty{})
 	if err != nil {
```

Comment:
when there is an error, shouldn't it return immediately instead? Then there are no more `lock`s and unneeded comparisons.

Reply:
Yes, right, fixed.